app/lib/features/events/sheets/create_event_sheet.dart

```diff
@@ -209,9 +209,9 @@ class _CreateEventSheetConsumerState extends ConsumerState<CreateEventSheet> {
       draft.utcEndFromRfc3339(utcEndDateTime);
       draft.descriptionText(description);
       final eventId = await draft.send();
-      final client = ref.read(clientProvider);
+      final client = ref.read(alwaysClientProvider);
```

Comment:
Good to have improved it, but why not using it everywhere else?

Reply:
I only tried to improve the problem that the integration tests failed with `null!` issues and replaced it where I found that pattern. We can extend that to other places, but it is only a softer version of it, so places where this throwing isn't acceptable, isn't good...